microxrce_agent.cpp

```diff
@@ -103,25 +104,37 @@ int main(int argc, char** argv)
     {
         std::cout << "UDP agent initialization... ";
         uint16_t port = parsePort(cl[1]);
-#ifdef _WIN32
```

Comment:
Discovery options should be disabled on Windows platforms until we have an implementation.

Reply:
Done!